ISISComputingGroup/ibex_gui
# Ticket2156: Allow setting macros when IOC first created

### Description of work

See ticket https://github.com/ISISComputingGroup/IBEX/issues/2156

### To test

* Add an IOC with a macro to a configuration
* Confirm that this macro is set on the IOC

System tests are included in https://github.com/ISISComputingGroup/ibex_system_tests/pull/37

---

#### Code Review


### Functional Tests


### Final Steps
